configs/9.0/packages/expat/sources

```diff
@@ -40,12 +42,45 @@ ATSRC_PACKAGE_BUNDLE=main_toolchain
 
 atsrc_get_patches ()
 {
-        at_get_patch \
-	https://hg.mozilla.org/releases/mozilla-esr31/raw-diff/2f3e78643f5c/parser/expat/lib/xmlparse.c \
-	978566c4144504d170869619661fbd7d || return ${?}
+	# Delete Makefile that generates man
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/a6bb3358ddae.patch \
+		bc61e4806ae531941b652bb56abb3249 || return ${?}
+
+	# Added man pages
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/f1a8200593de.patch \
+		0c71eb2bb1b6750a96c3c5181f67de3e || return ${?}
+
+	#Fix CVE-2012-6702 and CVE-2016-5300
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/95dbdf173f19.patch \
+		ea1a061d504d47aa726d09dc639db1fa || return ${?}
+
+	#Fix CVE-2016-4472
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/ca78e04e3e67.patch \
+		413e2b92eb8b6b35b8e22a1f56a2421e || return ${?}
+
+	#Fix CVE-2016-0718
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/be940daed613.patch \
+		1bd7c8c0414bd0a6f13acae171118adf || return ${?}
+
+	#Fix CVE-2016-9063 and CVE-2017-9233
+	at_get_patch \
+		https://github.com/racardoso/libexpat/commit/a39744e4ec1e.patch \
+		50c236fd76f8868a48468faff968fc9f || return ${?}
```

Comment:
The commit IDs listed in this patch are from libexpat itself and they apply without errors, so wouldn't it be better to just download them and use them directly:

```
at_get_patch \
  https://github.com/libexpat/libexpat/commit/d4f735b88d9932bd5039df2335eefdd0723dbe20.patch
  [...]
at_get_patch \
  https://github.com/libexpat/libexpat/commit/c4bf96bb51dd2a1b0e185374362ee136fe2c9d7f.patch
  [...]

[...]
patch -p1 < d4f735b88d9932bd5039df2335eefdd0723dbe20.patch
patch -p1 < c4bf96bb51dd2a1b0e185374362ee136fe2c9d7f.patch
```

Reply:
@gftg85 See the answer bellow.